python/torch_mlir_e2e_test/test_suite/index_put.py

```diff
@@ -818,3 +818,27 @@ def forward(self, input, index, value):
 def IndexPutHackedTwin3DIntAccumulateModule_basic(module, tu: TestUtils):
     module.forward(torch.randint(1000, (10, 8, 6)), torch.randint(4, (5, )),
                    torch.randint(1000, (5, 8, 6)))
+
+class IndexAddModule(torch.nn.Module):
+
+  def __init__(self):
+    super().__init__()
+
+  @export
+  @annotate_args([
+      None,
+      ([-1, -1, -1], torch.float32, True),
+      ([-1], torch.float32, True),
+      ([-1], torch.float32, True),
+      ([-1, -1], torch.float32, True),
```

Comment:
These annotations should match with the dtype and shape of the arguments passed in `module.forward()`